src/sentry/static/sentry/app/components/charts/utils.tsx

```diff
@@ -20,11 +21,7 @@ export const AREA_COLORS = [
   {line: '#422C6F', area: '#422C6F'},
 ];
 
-export type DateTimeObject = {
-  start: Date | null;
-  end: Date | null;
-  period?: string;
-};
+export type DateTimeObject = Partial<GlobalSelection['datetime']>;
```

Comment:
We have a few definitions of a DateTimeObject in the app, and they are all slightly different. I thought that normalizing on the GlobalSelection definition would be an improvement as that is generally what the values should be.